### ISSUE TITLE
Upgrade instructions/migrations from v1.x.x to v2.x.x

### ISSUE THREAD

**Issue by Alvie** [apioo/fusio#371]
Hi, I have started to use the Fusio 2.0.0 and have a good idea of some of the changes,

I was wondering if there are any instructions on upgrading / migrating the v1.x.x to 2.x.x in order to keep users etc in the database?

The users are the main thing that I am trying to port across as I can assign scopes etc later

**Comment by chriskapp** [comment]
Hi, so currently we have not provided a migration since there are many fundamental changes and also to make the new 2.0 installation faster. Maybe as workaround you can use the `system:export` command to export all existing users and use `system:import` on the new system.

**Comment by Alvie** [comment]
Thank you very much, I will try this and report my findings :)

**Comment by Alvie** [comment]
Hi, I have tried this but I am receiving a message that the command is not defined:
```
$ php bin/fusio system:export
```
```
                                           
  Command "system:export" is not defined.  
                                           
  Did you mean one of these?               
      deploy:export                        
      deploy:import                        
      system:check                         
      system:clean                         
      system:clear_cache                   
      system:cronjob_execute               
      system:log_rotate                    
      system:push                          
      system:register                      
      system:restore                       
      system:token                         
      system:user_add                      
                                           
```
I have tried deploy:export but that doesn't include the users (understandably). Am I using the command correctly? Or is there another workaround that you know of that might work? Thanks

**Comment by Alvie** [comment]
I have used `system:export` on the existing system which has created a JSON list of the users (amongst other things), I have extracted the users only and put them in a file called import.json in the root of the new system and used `deploy:import`:

```
$ php bin/fusio deploy:import import.json 

Import successful!
```
However, using `php bin/fusio user:list`, I cannot see the users there, nor can I see the users in the `dbname.fusio_user` table (viewed via PHPMyAdmin), the below is what my import.json file looks like (different scopes/dates though)
```json
{
    "user": [
      {
        "provider": 1,
        "status": 1,
        "name": "test2",
        "email": "test2@example.com",
        "scopes": [
            "abc",
            "xyz"
        ],
        "date": "2021-02-16T12:34:56Z"
      },
      {
        "provider": 1,
        "status": 1,
        "name": "test",
        "email": "test@example.com",
        "scopes": [
            "abc",
            "xyz"
        ],
        "date": "2021-12-16T12:34:56Z"
      }
    ]
  }
```

**Comment by Alvie** [comment]
As mentioned above, this unfortunately did not work. What I ended up doing is used the /backend/user (GET) route to get a list of the users, then iterated through each user (/backend/user/:id) to GET their status (now roleId with different numbers) and their scopes. With this information, I used the /consumer/register (POST) route to create the new user and then the /backend/user (PUT) route again to update the roleId and scopes.

Also, extracting the old SQL passwords from the 1.x.x database and applying to the relevant user in the 2.x.x database seems to have allowed the users to use the same credentials
